ansible/Vagrantfile

```diff
@@ -0,0 +1,90 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  config.vm.box = "ubuntu/bionic64"
+
+  config.vm.network "forwarded_port", guest: 9001, host: 9001
+  config.vm.network "forwarded_port", guest: 11001, host: 10001
+  config.vm.network "forwarded_port", guest: 22, host: 2222
+
+  config.vm.provision "ansible" do |ansible|
+    ansible.verbose = "v"
+    ansible.playbook = "volto-demo.yml"
+    ansible.inventory_path = "vbox_host.cfg"
+    # ansible.limit = "<name_of_your_host>"
+    # ansible.tags = "yourtag"
+  end
+
+  # config.vm.provider "virtualbox" do |v|
+  #   v.memory = 4000
+  #   v.cpus = 2
+  # end
+
+  # This is needed for change port configs
+  # config.ssh.insert_key = true
+  # config.ssh.guest_port = 2223
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  # config.vm.box_check_update = false
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+  # config.vm.network "forwarded_port", guest: 80, host: 8080
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  # config.vm.network "private_network", ip: "192.168.33.10"
+
+  # Create a public network, which generally matched to bridged network.
```

Comment:
```suggestion
  # Create a public network, which generally matches to a bridged network.
```

Reply:
Not mine, just test from the Vagrant init script.